UltraStar Play/ProjectSettings/ProjectSettings.asset

```diff
@@ -287,7 +385,7 @@ PlayerSettings:
   enableCrashReportAPI: 0
   cameraUsageDescription: 
   locationUsageDescription: 
-  microphoneUsageDescription: 
+  microphoneUsageDescription: This game needs access to the microphone. Highscores will be awarded for correctly singing along with the music.
```

Comment:
should probably be changed to `along "to" the music`

Reply:
👍

Reply:
@frankkienl could you please update your branch/fork and thus also update this pull request with this small change? Thanks!

Reply:
@frankkienl ping

Reply:
I would recommend to close this pull request. 

(Don't worry about the changes, they're all automatically generated when you select iOS target in Unity.)

We can work on iOS support later, when we've talked about how we will do deployment.

Reply:
@frankkienl maybe we can talk about this next weekend in Utrecht?

Reply:
Perfect